src/odemis/dataio/tiff.py

```diff
@@ -2613,3 +2624,121 @@ def _iterDirectories(tiff_file):
             tiff_file.ReadDirectory()
             dir_index += 1
             yield dir_index
+
+
+ 
+### FEI Metadata parsing
+def _parseINIMetadataAsDict(metadata: str, delimeter: str = "\r\n"):
+    """
+    Parses ini metadata as dictionary
+    metadata (str): metadata string as ini
+    delimeter (str): delimeter between rows (optional, default: "\r\n")
+    return (dict): metadata as dictionary
+    """
+    md = {}
+    for item in metadata.split(delimeter):
+        if item == "":
+            continue # skip blank lines
+        elif re.match(r"\[(.*?)\]", item):
+            subsystem =  item[1:-1] # rm brackets
+            md[subsystem] = {}
+        else:
+            k, v = item.split("=")   
+            md[subsystem][k] = v
+    return md
+
+
+def _convertFEItoOdemisMD(metadata):
+    """
+    Converts FEI metadata to odemis metadata format
+    metadata (str): metadata string as ini (FEI format)
+    return (dict): metadata as dictionary (odemis format)
+    """
+
+    # parse metadata as dictionary
+    fei_md = _parseINIMetadataAsDict(metadata)
+
+    # convert to odemis format
+    md = {}
+    
+    # general
+    datetime_str = fei_md["User"]["Date"] + " " + fei_md["User"]["Time"]
+    date = datetime.strptime(datetime_str, "%m/%d/%Y %I:%M:%S %p")
+    md[model.MD_ACQ_DATE] = date.timestamp()
+    
+    md[model.MD_HW_NAME] = fei_md["System"]["SystemType"]+"-"+fei_md["System"]["Dnumber"]    
+    md[model.MD_SW_VERSION] = fei_md["System"]["Software"]
+    md[model.MD_PIXEL_SIZE] = (float(fei_md["Scan"]["PixelWidth"]), 
+                                  float(fei_md["Scan"]["PixelHeight"]))
+    
+    md[model.MD_DWELL_TIME] = float(fei_md["Scan"]["Dwelltime"])
+    md[model.MD_DIMS] = "YX"
```

Comment:
Yes, on 2D data MD_DIMS is not set, as it's the default. It's only necessary for RGB because RGB images are 3D (Y, X, 3), and Odemis assumes by default that 3D is ZYX.